src/applications/vaos/sass/vaos.scss

```diff
@@ -1,7 +1,7 @@
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 @import "../../../platform/forms/sass/m-schemaform";
 
-.vaos-landing__icon {
+.vaos-landing__icon, .vaos-appt-list__icon {
```

Comment:
We could probably come up with a single class name for this. `vaos__option-list-icon` or something?